tests/test_exceptions.py

```diff
@@ -159,3 +80,10 @@ def test__custom_exception__contains_message_and_status_code_according_to_its_do
     assert issubclass(custom_exception, LambdaFWException)
     assert exception.message == message
     assert exception.status_code == int(code)
+
+
+def test__all_lbz_errors__returns_all_subclasses_of_lambda_fw_exception():
+    result = list(all_lbz_errors())
+
+    assert BadRequestError in result
+    assert ServerError in result
```

Comment:
You could also confirm that the list does not contain:
- LambdaFWException
- LambdaFWClientException
- LambdaFWServerException

They are skipped intentionally and we shouldn't change that in the future 😉